pkg/harvester/edit/loadbalancer.harvesterhci.io.ippool/PriorityRow.vue

```diff
@@ -0,0 +1,227 @@
+<script>
+import { _EDIT, _VIEW } from '@shell/config/query-params';
+import Select from '@shell/components/form/Select';
+import { NAMESPACE, MANAGEMENT, CAPI } from '@shell/config/types';
+import { mapGetters } from 'vuex';
+import { HCI } from '@pkg/harvester/types';
+
+export default {
+  components: { Select },
+
+  props: {
+    row: {
+      type:    Object,
+      default: () => ({}),
+    },
+
+    mode: {
+      type:    String,
+      default: _EDIT,
+    },
+
+    rows: {
+      type:    Array,
+      default: () => [],
+    },
+
+    idx: {
+      type:     Number,
+      required: true,
+    },
+  },
+
+  data() {
+    return { value: '' };
+  },
+
+  computed: {
+    ...mapGetters(['isSingleVirtualCluster']),
+    ...mapGetters(['currentCluster']),
+
+    isView() {
+      return this.mode === _VIEW;
+    },
+
+    showRemove() {
+      return !this.isView;
+    },
+
+    namespaceOptions() {
+      const namespaces = this.$store.getters['harvester/all'](NAMESPACE);
+
+      const out = namespaces.filter((namespace) => {
```

Comment:
Do we need to filter out some system namespaces here ?

Reply:
Done